scraper/src/documentation_spider.py

```diff
@@ -76,19 +81,20 @@ def __init__(self, config, algolia_helper, strategy, *args, **kwargs):
     def start_requests(self):
 
         # We crawl according to the sitemap
-        for i in self.start_requests_sitemap():
-            yield i
+        if self.sitemap_urls:
+            for i in self.start_requests_sitemap():
```

Comment:
can we renam `i` to `request`. It will be easier to understand what we are dealing with